plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseClient.java

```diff
@@ -299,7 +300,7 @@ protected String createTableSql(RemoteTableName remoteTableName, List<String> co
         formatProperty(ClickHouseTableProperties.getOrderBy(tableProperties)).ifPresent(value -> tableOptions.add("ORDER BY " + value));
         formatProperty(ClickHouseTableProperties.getPrimaryKey(tableProperties)).ifPresent(value -> tableOptions.add("PRIMARY KEY " + value));
         formatProperty(ClickHouseTableProperties.getPartitionBy(tableProperties)).ifPresent(value -> tableOptions.add("PARTITION BY " + value));
-        ClickHouseTableProperties.getSampleBy(tableProperties).ifPresent(value -> tableOptions.add("SAMPLE BY " + value));
+        ClickHouseTableProperties.getSampleBy(tableProperties).ifPresent(value -> tableOptions.add("SAMPLE BY " + quoted(value)));
```

Comment:
Could you quote other properties as well? 

Ideally, it would be better to check if specified columns exist in the table definition and then avoid executing queries if not found. This can be a follow-up PR.

Reply:
> Could you quote other properties as well?

All other properties quoted inside formatProperty method

Reply:
> Could you quote other properties as well?

All other properties quoted inside formatProperty method

Reply:
I was missing that, thanks!